Production/govuk_ios/Resources/Assets.xcassets/OnboardingImages/onboarding_placeholder_screen_1.imageset/Contents.json

```diff
@@ -1,9 +1,8 @@
 {
   "images" : [
     {
-      "filename" : "onboarding_placeholder_screen_1.png",
-      "idiom" : "universal",
-      "scale" : "1x"
+      "filename" : "onboarding_placeholder_screen_1.svg",
```

Comment:
As this change was to just replace onboarding images I haven't touched `placeholder` naming. Unsure if images were initially called `placeholder` because they were temporary? They are referred to as `placeholder` elsewhere in the project so I haven't touched the naming.